docs/src/styles/global.css

```
@@ -0,0 +1,853 @@
+  --muted: #aaa99f;
+  --dim: #80847d;
+  --paper: #0d0f0f;
+  --panel: #151818;
+  --panel-2: #1b1e1d;
+  --line: #2a2e2c;
+  --acid: #d8ff5f;
+  --aqua: #63e5d5;
+  --orange: #ff9f5a;
+  --code: #111414;
+  --shadow: rgba(0, 0, 0, 0.4);
+  --serif: Georgia, "Times New Roman", serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)